recipes/laxpy/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "laxpy" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/brycefrank/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 9b465296e9a4f6a33415048e8743faff00e6db9af3ac94e411c8749b0e92769b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - geopandas
+    - laspy
+    - lastools
+    - numba
+    - numpy
+    - shapely
+    - python >=3.6
+    - pip
```

Comment:
In ``host`` section I think you just need ``laspy``, ``python`` and ``pip``

Reply:
I was hoping to get some advice there. I will make the modifications soon.